source/rust.txt

```diff
@@ -37,35 +37,71 @@ See `Installation <https://github.com/mongodb/mongo-rust-driver#Installation>`__
 Connect to MongoDB Atlas
 ------------------------
 
+Select from the :guilabel:`Sync` or :guilabel:`Async` tabs below for
+corresponding connection code samples.
+
 .. include:: /includes/atlas-connect-blurb.rst
 
-.. code-block:: rust
+.. tabs::
+
+   .. tab:: Sync
+      :tabid: rust-sync
+
+      .. code-block:: rust
+
+         // Be sure to set default-features=false and enable the "sync" feature in your Cargo.toml
+         use mongodb::{bson::doc, sync::Client};
+
+         fn main() -> mongodb::error::Result<()> {
+             // Get a handle to the deployment.
+             let client = Client::with_uri_str(
+                 "mongodb+srv://<username>:<password>@<cluster-url>/test?w=majority",
+             )?;
+
+             client
```

Comment:
Maybe add a comment explaining why we ping the admin db? Is this required to connect or just something we're doing for demo purposes?

Reply:
Not sure it's necessary, will ask Patrick.

Reply:
The `Client` will be connecting in the background, so the ping isn't necessary. Running a ping here is basically a test that the background connections are succeeding, but the same thing could be achieved via just the subsequent `list_database_names` call. Perhaps it makes sense to just remove the ping and associated `println` then if it adds confusion.